changelog.md

```diff
@@ -1,4 +1,8 @@
 # Change Log
+## 3.1.4
+**Fixes**
+ * properly handle ForbiddenAccess for Include
```

Comment:
This description is too vague I think since "proper" may be-- more or less-- a matter of opinion. Maybe something like:

```
Changed handling of including forbidden collections. Instead of throwing a ForbiddenAccess exception, we filter the included collection.
```

Reply:
Of course only my opinion matters 😄   But will change the text.